### PR TITLE
allow more general single stat in coinc statistic

### PR DESCRIPTION
**Pull request by ahnitz**
The expfitfgbg statistic was somewhat hardcoded in the way it chose a single detector statistic, now any can be used in its construction.

### CODE REVIEW

File: pycbc/events/stat.py [hunk 1]
```diff
@@ -1546,7 +1543,8 @@ def coinc_lim_for_thresh(self, s, thresh, limifo,
         """
 
         # Safety against subclassing and not rethinking this
-        allowed_names = ['ExpFitSGFgBgNormStatistic']
+        allowed_names = ['ExpFitSGFgBgNormStatistic',
+                         'ExpFitSGPSDFgBgNormBBHStatistic']
```
Comment (GarethDaviesGW) [inline pycbc/events/stat.py:1547]: I'll have to take a bit more time to look through this, but this looks like PSD and BBH are mutually required, is that correct/desired?

Comment (ahnitz) [reply pycbc/events/stat.py:1547]: It is not actually correct anymore, the names are somewhat misleading atm. Would you suggest renaming? I could do that.

Comment (ahnitz) [reply pycbc/events/stat.py:1547]: i.e. you can use any single stat now there in either of the multi-det coinc classes

Comment (GarethDaviesGW) [reply pycbc/events/stat.py:1547]: In that case removing both "SG" and "PSD" should make sense

Comment (ahnitz) [reply pycbc/events/stat.py:1547]: OK, I can do that. I'll need to update the test config files as well. And you all should note that for your internal configs as well if  using these classes.

Comment (ahnitz) [reply pycbc/events/stat.py]: @GarethDaviesGW Ok, I've renamed everything by dropping the sg / psd parts everywhere.

**Review by GarethDaviesGW — APPROVED** [review]

**Pull request merged**